pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -14,38 +14,66 @@ import (
 	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
 )
 
-type CIDataClient interface {
```

Comment:
separated this client into the bits used by specific pieces so that I could figure out how broad the scope of change was for various jobrun tables.